core/src/main/java/io/confluent/rest/errorhandlers/NoJettyDefaultStackTraceErrorHandler.java

```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright 2022 Confluent Inc.
- */
-
-package io.confluent.rest.errorhandlers;
-
-import java.io.IOException;
-import jakarta.servlet.RequestDispatcher;
-
-import org.eclipse.jetty.ee10.servlet.ServletContextRequest;
-import org.eclipse.jetty.http.HttpStatus;
-import jakarta.servlet.http.HttpServletRequest;
-import jakarta.servlet.http.HttpServletResponse;
-import org.eclipse.jetty.ee10.servlet.ErrorHandler;
-
-public class NoJettyDefaultStackTraceErrorHandler extends ErrorHandler {
-  public NoJettyDefaultStackTraceErrorHandler() {
-    super();
-    setShowServlet(false);
```

Comment:
this `setShowServlet` is not used in Jetty 12 org.eclipse.jetty.server.handler.ErrorHandler . We replace it with  `setShowCauses(!suppressErrors);` and `setShowStacks(!suppressErrors);` in `StackTraceErrorHandler.java`